src/core.js

```diff
@@ -71,30 +70,27 @@ jQuery.parseJSON = function() {
 	return JSON.parse.apply( null, arguments );
 };
 
-jQuery.isNumeric = function( val ) {
+migrateWarnFunc( jQuery, "isNumeric", function( obj ) {
```

Comment:
Do we want to use the old implementation, not the latest one?

Reply:
There was already a jQuery 3.x warning for a very esoteric change. Since we're now deprecating the whole method now it seemed easier to roll back to the old implementation which is probably what people who are using it really want.